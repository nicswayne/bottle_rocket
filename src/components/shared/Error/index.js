// @flow

import React from 'react';

const ErrorComponent = ({ error }: { error: string }) => (
    <div>
        <h5>{error}</h5>
    </div>
);

export default ErrorComponent;
