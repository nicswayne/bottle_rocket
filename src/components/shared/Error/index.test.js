import React from 'react';
import { shallow, render } from 'enzyme';

import Error from '.';

it('Error component should render without crashing.', () => {
    shallow(<Error />);
});

it('Error component should match snapshot.', () => {
    const wrapper = render(<Error error="Error message." />);

    expect(wrapper).toMatchSnapshot();
});
