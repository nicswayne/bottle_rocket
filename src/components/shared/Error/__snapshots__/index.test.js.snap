// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should match snapshot. 1`] = `
<div
  class="error"
>
  <h3>
    Error message.
  </h3>
  Please try again.
  <button
    class="reload text"
  >
    reload
  </button>
</div>
`;
